include/vcpkg/base/messages.h

```diff
@@ -1292,6 +1354,7 @@ namespace vcpkg
                     "No packages are installed. Did you mean `search`?");
     DECLARE_MESSAGE(NoLocalizationForMessages, (), "", "No localized messages for the following: ");
     DECLARE_MESSAGE(NoRegistryForPort, (msg::package_name), "", "no registry configured for port {package_name}");
+    DECLARE_MESSAGE(NugetPackageExported, (msg::path), "", "NuGet package exported at: {path}");
```

Comment:
I think this should reuse `CreatedNuGetPackage`. Or do the "creating NuGet package {path}" transform I mentioned before and fix that place too.